lib/watir/cookies.rb

```diff
@@ -119,15 +119,5 @@ def load(file = '.cookies')
         add(c.delete(:name), c.delete(:value), c)
       end
     end
-
-    private
-
-    def to_time(time)
-      if time.respond_to?(:to_time)
```

Comment:
Do you know why this was needed?

Reply:
I spent a solid half hour trying to figure out what class instance would respond to year / month methods and not to_time. Then I realized that this is only acting on information coming back from selenium, so we know that what the object will be nil or date time

Reply:
🤷‍♂️